src/Makefile

```diff
@@ -1,271 +1,155 @@
 #
-# Change all boolean variables to lowercase
+#  Change all boolean variables to lowercase
 #
 #-----------------------------------------------------------------------------
-USE_OPENMP_L =  $(shell echo $(USE_OPENMP) | tr A-Z a-z)
-USE_MPI_L =  $(shell echo $(USE_MPI) | tr A-Z a-z)
-USE_DOUBLE_L =  $(shell echo $(USE_DOUBLE) | tr A-Z a-z)
-USE_OPENACC_L =  $(shell echo $(USE_OPENACC) | tr A-Z a-z)
-DEBUG_L =  $(shell echo $(DEBUG) | tr A-Z a-z)
+USE_OPENMP_L  = $(shell echo $(USE_OPENMP) | tr A-Z a-z)
+USE_MPI_L     = $(shell echo $(USE_MPI) | tr A-Z a-z)
+USE_DOUBLE_L  = $(shell echo $(USE_DOUBLE) | tr A-Z a-z)
+USE_OPENACC_L = $(shell echo $(USE_OPENACC) | tr A-Z a-z)
+DEBUG_L       = $(shell echo $(DEBUG) | tr A-Z a-z)
+
+#-----------------------------------------------------------------------------
+#  initialize some options as empty
+#-----------------------------------------------------------------------------
+DM            =
+OMP           =
+DP            =
+OPTS          =
+CPP           =
+OUTPUTINC     =
+OUTPUTLIB     =
+OUTPUTOPT     =
+LINKOPTS      =
 
 #-----------------------------------------------------------------------------
 #                      NETCDF SECTION
 #  (uncomment the following four lines if you want netcdf output capability)
 #   (also, make sure the paths to netcdf files are correct for your machine)
 #              (NOTE: Don't change lines 3 and 4!)
 #  Note:  you may need to remove -DNCFPLUS if you use an old version of netcdf
-#
-OUTPUTINC = -I$(NETCDFBASE)/include
-OUTPUTLIB = -L$(NETCDFBASE)/lib
-OUTPUTOPT = -DNETCDF -DNCFPLUS
-LINKOPTS  = -lnetcdf -lnetcdff
 #-----------------------------------------------------------------------------
+OUTPUTINC     += -I$(NETCDFBASE)/include
+OUTPUTLIB     += -L$(NETCDFBASE)/lib
+OUTPUTOPT     += -DNETCDF -DNCFPLUS
+LINKOPTS      += -lnetcdf -lnetcdff
 
-
 #-----------------------------------------------------------------------------
-#                     HARDWARE SECTION
-#-- Choose the appropriate architecture, and uncomment all lines 
-#-- in that section.  See "README.compile" for more information.
+#                    HARDWARE SECTION
+#  Do not comment/uncomment the options below manually.
+#  Specify the boolean variables above to switch configurations.
+#  If a compiler flag is missing, issue a PR in CM1 Repo.
+#-----------------------------------------------------------------------------
 
-ifeq (${FC},ifort)
 #-----------------------------------------------------------------------------
-ifeq (${USE_MPI_L},false)
-ifeq (${USE_OPENMP_L},false)
-#  single processor, Intel compiler 
-#      (eg, NCAR's yellowstone/caldera/cheyenne)
-override FC   = ifort
-OPTS = -g -O3 -xHost -ip -assume byterecl -fp-model precise -ftz -no-fma -init=snan,arrays -traceback
-CPP  = cpp -C -P -traditional -Wno-invalid-pp-token -ffreestanding
+#  NVHPC compiler
+#-----------------------------------------------------------------------------
+ifeq (${FC},nvfortran)
```

Comment:
@sjsprecious  While this makefile is more readable versus the previous version I think there is still room for improvement.  In particular, I think that indents could be used so as to make the if structure much easier to understand.

Reply:
@johnmauff Thanks for the comments. Just add the indents.

Reply:
Thanks, this looks good